lib/index.js

```diff
@@ -8,11 +8,17 @@ const path = require('path');
 const sass = require('node-sass');
 const argv = require('argv-parse');
 const verify = require('./verify');
+const {winston} = require('./logging');
 
 const thenify_all = require('thenify-all');
 const fs = thenify_all(require('fs-extra'), {}, ['readFile', 'writeFile', 'readdir', 'copy']);
 const tar = require('tar');
 
+process.on('unhandledRejection', ((reason, promise) => {
+    winston.error("Unhandled promise rejection. Reason: ", reason);
+    process.exitCode = 1;
+}));
```

Comment:
If you're putting a hook on the process, you can do that in the logging module to keep things self-contained

Reply:
Followup: you can do process.once() maybe? Not sure how it will deal with multiple logging module imports.

Reply:
Sounds good. I am also working on better debugging of assets that don't pass the validations. Currently we get a obscure validation error, and is hard to know what caused it.